README.md

```diff
@@ -1,33 +1,33 @@
 [![semaphore ci build status](https://semaphoreci.com/api/v1/cockpit/cockpit/branches/master/badge.svg)](https://semaphoreci.com/cockpit/cockpit) <br />
-[![Fedora 23 build status](https://fedorapeople.org/groups/cockpit/status/fedora-23.svg)](https://fedorapeople.org/groups/cockpit/status/fedora-23.html) <br />
-[![Fedora 24 build status](https://fedorapeople.org/groups/cockpit/status/fedora-24.svg)](https://fedorapeople.org/groups/cockpit/status/fedora-24.html) <br />
-[![Fedora 23 Atomic Host status](https://fedorapeople.org/groups/cockpit/status/fedora-atomic.svg)](https://fedorapeople.org/groups/cockpit/status/fedora-atomic.html) <br />
-[![Fedora 24 Testing status](https://fedorapeople.org/groups/cockpit/status/fedora-testing.svg)](https://fedorapeople.org/groups/cockpit/status/fedora-testing.html) <br />
-[![RHEL 7 build status](https://fedorapeople.org/groups/cockpit/status/rhel-7.svg)](https://fedorapeople.org/groups/cockpit/status/rhel-7.html) <br />
-[![Debian Unstable build status](https://fedorapeople.org/groups/cockpit/status/debian-unstable.svg)](https://fedorapeople.org/groups/cockpit/status/debian-unstable.html)
 
 # Cockpit
 **A remote manager for GNU/Linux servers**
 
-### About
+[cockpit-project.org](http://cockpit-project.org/)
+
 Cockpit is an interactive server admin interface. It is easy to use and very light weight.
+Cockpit interacts directly with the operating system from a real Linux session in a browser.
+
+### Using Cockpit
 
-#### Easy to use
-Cockpit is perfect for new sysadmins, allowing them to easily perform simple tasks such as storage administration, inspecting journals and starting and stopping services.
+You can [install Cockpit](http://cockpit-project.org/running.html) on many Linux operating
+systems including Debian, Fedora and RHEL.
 
-#### No interference
-Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
+Cockpit makes Linux discoverable, allowing sysadmins to easily perform tasks such as starting
+containers, storage administration, network configuration, inspecting logs and so on.
 
-#### Multi-server
-You can monitor and administer several servers at the same time. Just add them with a single click and your machine will look after its buddies.
+Jumping between the terminal and the web tool is no problem. A service started via Cockpit
+can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen
+in the Cockpit journal interface.
+
+You can monitor and administer several servers at the same time. Just add it easily and your
```

Comment:
"Just add it easily" is unclear. How about something like `On the Cockpit dashboard, you can easily add other machines with Cockpit installed that are accessible via ssh and your server will look after its buddies.`